include/ros1_bridge/factory.hpp

```diff
@@ -45,7 +45,7 @@ class Factory : public FactoryInterface
     const std::string & topic_name,
     size_t queue_size)
   {
-    rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_sensor_data;
+    rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_default;
```

Comment:
Since this uses the default it can just omit the profile argument.

Same below.